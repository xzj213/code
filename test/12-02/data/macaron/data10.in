10
2720 0 0 3692 3692 2720 1647 0 2720 1647
3692 1647 1647 3589 0 0 0 1647 2720 0
2720 1647 2720 0 0 1337 3692 2720 2720 3692
3692 0 1647 0 1647 2720 1647 0 1647 0
1647 3692 3692 1647 2720 3692 3692 1647 0 3692
2720 0 0 3692 1647 3692 2720 2720 3692 1647
3692 3692 2720 1647 1647 1647 0 162 2720 3692
3692 3692 3692 2720 1647 2720 2720 2720 2720 1647
3692 2720 2720 0 3692 2720 2720 1801 0 2720
3692 3692 2720 2720 3692 3692 1647 1473 0 1647
