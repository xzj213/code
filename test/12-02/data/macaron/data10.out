0 105 145 147 162 203 248 250 269 271 318 343 356 358 412 501 513 560 562 601 603 616 675 714 831 854 925 940 942 965 967 1012 1031 1076 1078 1117 1119 1134 1189 1228 1337 1360 1435 1448 1450 1473 1475 1522 1542 1647 1685 1687 1700 1741 1788 1790 1801 1803 1848 1873 1888 1890 1946 2035 2051 2154 2192 2194 2209 2248 2297 2299 2316 2318 2365 2388 2405 2407 2463 2550 2562 2609 2611 2648 2650 2667 2720 2761 2876 2901 2974 2989 2991 3012 3014 3063 3076 3125 3127 3164 3166 3181 3238 3279 3386 3411 3480 3497 3499 3520 3522 3569 3589 3692 3732 3734 3751 3790 3837 3839 3848 3850 3899 3922 3937 3939 3993 4080 
